### DISCUSSION TITLE
Customize look & feel

### DISCUSSION THREAD

**Discussion by henrycunh** [stoplightio/elements#1261] [Ideas]
Hey there (:

My team is researching this solution, and so far we have found no easier way to customize the looks of it than to fork the project and directly meddle with the implementation. 

Are any plans for easing the experience of customization in place?

Really love what you guys did here, what an amazing tool!

**Comment by Lee4Gitr** [comment]
It's unfortunate that their devs haven't responded to this yet, as its 2 years old and the highest rated request so far.

**Reply by henrycunh** [reply]
I've been using this https://github.com/scalar/scalar

Kinda perfect ngl
